distribution/linux/openrct2-scenario.desktop

```diff
@@ -0,0 +1,11 @@
+# http://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.0.html
+
```

Comment:
Minor nitpick, but the other two `.desktop` files have no newline here. It would slightly more consistent to drop it.